### PR TITLE
Document max value for GET event users limit param

### PR DESCRIPTION
**Pull request by lts20050703**
Closes: #1877

### CODE REVIEW

File: src/helpers/guilds/scheduledEvents/getScheduledEventUsers.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { GetScheduledEventUsers } from "../../../types/guilds/scheduledEvents.ts
 import { GuildMember } from "../../../types/members/guildMember.ts";
 import { User } from "../../../types/users/user.ts";
 import { Collection } from "../../../util/collection.ts";
+import { validateLength } from "../../../util/mod.ts";
```
Comment (itohatweb) [inline src/helpers/guilds/scheduledEvents/getScheduledEventUsers.ts:7]: Why this import when its not used?

File: src/types/guilds/scheduledEvents.ts [hunk 2]
```diff
@@ -124,9 +124,9 @@ export interface GetScheduledEvents {
 }
 
 export interface GetScheduledEventUsers {
-  /** how many users to receive from the event. Defaults to 100. */
+  /** number of users to return (up to maximum 100), defaults to 100. */
```
Comment (itohatweb) [inline src/types/guilds/scheduledEvents.ts:127]:
```suggestion
  /** number of users to return (up to maximum 100), defaults to 100 */
```

File: src/types/guilds/scheduledEvents.ts [hunk 3]
```diff
@@ -124,9 +124,9 @@ export interface GetScheduledEvents {
...
   limit?: number;
-  /** Whether to also have member objects provided. Defaults to false. */
+  /** whether to also have member objects provided, defaults to false. */
```
Comment (itohatweb) [inline src/types/guilds/scheduledEvents.ts:129]:
```suggestion
  /** whether to also have member objects provided, defaults to false */
```

**Review by itohatweb — CHANGES_REQUESTED** [review]

File: src/helpers/guilds/scheduledEvents/getScheduledEventUsers.ts [hunk 4]
```diff
@@ -25,10 +26,13 @@ export async function getScheduledEventUsers(
 ): Promise<
   Collection<bigint, DiscordenoUser> | Collection<bigint, { user: DiscordenoUser; member: DiscordenoMember }>
 > {
-  // TODO: validate limit
+  if (options?.limit && options.limit > 100) {
```
Comment (Skillz4Killz) [inline src/helpers/guilds/scheduledEvents/getScheduledEventUsers.ts:29]: move to plugin

File: src/helpers/guilds/scheduledEvents/getScheduledEventUsers.ts (hunk 1 above)
Comment (lts20050703) [reply src/helpers/guilds/scheduledEvents/getScheduledEventUsers.ts:7]: oops I was going to use it then realize limit is a number and not a string so yeah let me remove it real quick

File: src/types/guilds/scheduledEvents.ts (hunk 3 above)
Comment (lts20050703) [reply src/types/guilds/scheduledEvents.ts:129]: lol ok

**Review by Skillz4Killz — CHANGES_REQUESTED** [review]

File: src/helpers/guilds/scheduledEvents/getScheduledEventUsers.ts (hunk 4 above)
Comment (lts20050703) [reply src/helpers/guilds/scheduledEvents/getScheduledEventUsers.ts:28]: Sorry can you elaborate? What do you mean "move to plugin" ? And what action should I take?

**Review by itohatweb — APPROVED** [review]

**Pull request merged**